doc/wasmoo.rst

```diff
@@ -101,4 +102,4 @@ By default, they are generated when using the ``dev`` build profile and are not
 The behavior can explicitly be specified in an ``env`` stanza (see :doc:`/reference/dune/env`)
 or per executable inside ``(wasm_of_ocaml (sourcemap ...))`` (see :doc:`/reference/dune/executable`)
 
-.. _wasm_of_ocaml: https://github.com/ocaml-wasm/wasm_of_ocaml
+.. _wasm_of_ocaml: https://github.com/ocsigen/js_of_ocaml
```

Comment:
With the new intro paragraph I think it's possible to remove the autolink on `wasm_of_ocaml` (remove the trailing underscore) and remove this line